catalog/transformer/catalog-transformer-generic-impl/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0"
+           xsi:schemaLocation="http://www.osgi.org/xmlns/blueprint/v1.0.0
+           http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd
+           http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0
+           http://aries.apache.org/schemas/blueprint-cm/blueprint-cm-1.1.0.xsd">
+
+
+    <type-converters>
+        <bean id="listConverter" class="ddf.catalog.util.impl.ListConverter"/>
+    </type-converters>
+
+    <bean id="saxEventHandlerFactoriesList" class="ddf.catalog.util.impl.SortedServiceList"/>
+    <reference-list id="saxEventHandlerFactories"
+                    interface="ddf.catalog.transformer.generic.xml.api.SaxEventHandlerFactory"
+                    availability="optional">
+        <reference-listener ref="saxEventHandlerFactoriesList" bind-method="bindPlugin"
+                            unbind-method="unbindPlugin"/>
+    </reference-list>
+
+
+    <cm:managed-service-factory id="XMLInputTransformerFactory"
+                                factory-pid="XMLInputTransformer"
+                                interface="ddf.catalog.transform.InputTransformer">
+
+        <cm:managed-component
+            class="ddf.catalog.transformer.generic.xml.lib.XMLInputTransformer"
+            init-method="init" destroy-method="destroy">
+        <!--Defines the methods to be invoked when the OpenSearch Source
+            instance is created and when it is deleted. The init method is called after
+            all of the setter methods have been called. -->
+        <property name="id" value="xml"/>
+        <property name="title"
+                  value="Base XML transformer"/>
+        <property name="organization" value="Codice"/>
+        <property name="saxEventHandlerFactories" ref="saxEventHandlerFactoriesList"/>
+        <property name="saxEventHandlerConfiguration">
+            <list>
+                <value>XML_Handler</value>
+            </list>
+        </property>
+
+        <cm:managed-properties persistent-id=""
```

Comment:
Why do you need this extra managed-properties element? This is already a managed-component.